firebase.json

```
@@ -221,6 +221,7 @@
       { "source": "/tools/dart2aot", "destination": "/tools/dart-compile", "type": 301 },
       { "source": "/tools/dart2native", "destination": "/tools/dart-compile", "type": 301 },
       { "source": "/tools/dartanalyzer", "destination": "/tools/dart-analyze", "type": 301 },
+      { "source": "/tools/dartdoc", "destination": "/tools/dart-doc", "type": 301 },
       { "source": "/tools/dartdocgen{,/**}", "destination": "https://github.com/dart-lang/dartdoc#dartdoc", "type": 301 },
```

What do you think about updating this redirect as well? 

```suggestion
      { "source": "/tools/dartdocgen{,/**}", "destination": "/tools/dart-doc", "type": 301 },
```

With the recent changes to the dartdoc README, the old link wouldn't jump to the README anymore. If you still want it to point to the GitHub repo, then you can point it to: https://github.com/dart-lang/dartdoc#dart-documentation-generator